app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,43 @@
+class VotesController < ApplicationController
+
+
+
+  def new
+    @vote = Vote.new
+  end
+
+
+  def create
+    p "Over Here!!!"
+    p "#{params[:test]}"
+    @question = params[:question_id]
+    @answer = params[:answer_id]
+
+    if @question    
+      if params[:like] == "yes"
+        @vote = Vote.create!(user_id: session[:user_id], votable_id: params[:question_id], votable_type: "Question", point: 1)
+      elsif params[:like] == "no"
```

Comment:
create! will throw an error here if it fails. It might be better to use the non-bang version and handle the case where it fails in your program flow control, rather than throwing an error.